docs/docs/examples/src/callbacks/onClick.js

```diff
@@ -0,0 +1,12 @@
+var module = pathfora.Message({
+  id: 'onClick-callback',
+  layout: 'button',
+  msg: 'Click Me!',
+  onClick: function (event, payload) {
+    console.log('OnClick Callback: ');
```

Comment:
ultra mega super nitpicky but could just do

`console.log('Onclick Callback', event, payload)` for any of these with three console.logs